style/style.css

```diff
@@ -0,0 +1,87 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+* {
+  overflow-x: hidden;
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  border: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+.container {
+  margin: 0 auto;
+  width: 85%;
+}
+
+header {
+  display: flex;
+  padding: 2rem 3rem;
+  justify-content: space-between;
+  align-items: center;
+  background: #3c3a39;
+  width: 100%;
+  margin: 0 auto;
+}
+
+header a {
+  color: white;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.main-section {
+  position: absolute;
+  width: 100%;
+  background: #1c1a19;
+}
+
+main {
+  background-image: url(/images/illustration.svg);
+  background-repeat: no-repeat;
+  background-size: 156px;
+  background-position-x: 161px;
+  background-position-y: 250px;
+  height: 40rem;
+  position: relative;
+}
```

Comment:
- [ ] Great job implementing requested changes by the previous reviewer. However, there's little stuf you still need to work on. At this moment, the background image on the page is not properly position, not responsive. Kindly work on it to make it appear as we have it on the figma design.

### Your Design
[bg.webm](https://user-images.githubusercontent.com/26597432/228916396-ece3cabc-add9-4c51-8090-408af08895f3.webm)

### Figma Design
![temp-bg](https://user-images.githubusercontent.com/26597432/228916446-fea16337-8c14-4a22-abed-be74e552e36f.png)

Reply:
I've positioned the background image properly and made it responsive.